README.md

```diff
@@ -54,6 +54,96 @@ Model::search('query')
 ```
 
 
+## `with`
```

Comment:
Thanks for the doc. 👍 
If you want to get it merged quickly, you can create a separate PR 👌